crates/core/src/host/host_controller.rs

```diff
@@ -106,12 +107,14 @@ pub struct HostController {
 
 struct HostRuntimes {
     wasmtime: WasmtimeRuntime,
+    v8: V8Runtime,
 }
 
 impl HostRuntimes {
     fn new(data_dir: Option<&ServerDataDir>) -> Arc<Self> {
         let wasmtime = WasmtimeRuntime::new(data_dir);
-        Arc::new(Self { wasmtime })
+        let v8 = V8Runtime::new();
+        Arc::new(Self { wasmtime, v8 })
```

Comment:
The way this is structured, we're always running V8, even when unused. Do we want to consider lazily initializing it?